packages/website/pages/docs/reference.mdx

```diff
@@ -1 +1,19 @@
-Handy reference documentation for Taiko.
+## Handy reference documentation for Taiko.
+
+### [Contract Documentation](reference/contract-documentation)
+
+This section contains the documentation related to different smart contracts deployed on the blockchain. It includes information about the contract's functionality, usage, and interfaces. The documentation also covers the technical details, such as the contract's bytecode, source code, and compiler version.
+
+### [Contract Addresses](reference/contract-addresses)
+
+This section provides the list of contract addresses deployed on the blockchain. It includes information about the contract's name, version, and the network it's deployed on. These addresses are crucial for interacting with the contracts using the various blockchain tools and libraries.
+
+### [Protocol Constants](reference/protocol-constants)
+
+This section provides the protocol constants that are fundamental to the blockchain's operation. These constants include block gas limits, chain IDs, EVM opcodes, and other network parameters. Understanding these constants is essential for building applications on the blockchain.
```

Comment:
can you please remove the sentences like "this is essential / crucial for building applications on the blockchain". in contract addresses, protocol constants, and rpc configuration.

because of docs philosophy here: https://github.com/taikoxyz/taiko-mono/blob/main/CONTRIBUTING.md#philosophy (minimum viable documentation)